examples/config/config

```diff
@@ -258,14 +260,20 @@ set history_disable_easter_egg 1
 @cbind  w            = event REQ_NEW_WINDOW
 
 # Page movement binds
-@cbind  j            = scroll vertical 20
-@cbind  k            = scroll vertical -20
-@cbind  h            = scroll horizontal -20
-@cbind  l            = scroll horizontal 20
+@cbind  j            = scroll vertical 100
+@cbind  J            = scroll vertical 50
```

Comment:
Eh, j already moves in little jumps as is. I don't think making it this much larger and making *capital* J moves in half that makes much sense.